### DISCUSSION TITLE
可以增加文件上传接口吗？

### DISCUSSION THREAD

**Discussion by tobycroft** [Mrs4s/go-cqhttp#832] [Q&A]
就是发送图片，这个我记得最早在Coolq那边是使用文件上传接口，例子那边是先上传图片再发送图片的hash，然后在gcq这边我看了应该要使用cq码来发送，但是总感觉是少了一个上传的接口，用的不是太会，文档上虽然将file注明了，但是文件的路径需要如何填写比较模糊

所以我想这块不知道能不能直接做成上传文件的，这样就没有对接难度了

**Comment by Mrs4s** [comment]
可以考虑保留一个直接上传的接口
